### ISSUE TITLE
pastel 0.2.2 - unrecognized escape sequence when mode is HumanReadable

### ISSUE THREAD

**Issue by antonstefanov** [facebookexperimental/reason-native#219]
I am using `Pastel.setMode(HumanReadable);` in tests for nice snapshots
(unrelated to the issue, but adding for reference:
https://github.com/antonstefanov/rarg/blob/master/tests/__tests__/rarg/Rarg_ComponentsHelp_test.re
and
https://github.com/antonstefanov/rarg/blob/master/tests/__snapshots__/RargComponentsHelp_Help.6ded9ba6.0.snapshot)

However with pastel version `0.2.2` the following snippet:
```re
Pastel.setMode(HumanReadable);
let hello = () => <Pastel bold=true> "<hello>" </Pastel>;
hello();
```
produces the following error:
```
Fatal error: exception Invalid_argument("unrecognized escape sequence '<hello>'.")
```

It is not reproducible in version `0.2.1`

**Comment by bandersongit** [comment]
Thanks for filing and sorry you ran into this. It almost certainly has to deal with some recent internal changes to the Pastel parser and will be fixed. In the short term I'd recommend using 0.2.1.

**Comment by antonstefanov** [comment]
👌 all good, it was an easy fix

thanks for Rely btw, the Jest familiarity made it much easier to transition some apps to Reason Native

**Comment by bandersongit** [comment]
Glad you've been enjoying Rely, I just released Pastel 0.2.3 which includes this fix
